docs/getting-started.md

```diff
@@ -19,7 +19,57 @@ You will also need JDK 17.
 If your application is on Spring Boot 2.7, you will need to use the 5.5.x release train of the DGS framework.
 If your application is on Spring Boot 2.6, you will need to use 5.4.x or earlier.
 
+
```

Comment:
A couple thoughts on this section:
- `DGS Framework now using Spring GraphQL!` feels like an announcement, which I find tend not to age well in docs, unless they're very carefully scoped to things like release notes.
- Reading through I think it could be easy to get confused about which of these sections actually apply. Perhaps having a "Install Dependencies" header which can explain the difference, and then demoting spring-graphql and classic sections to a `###` header?